futures/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,11 @@
+name: Scala 3 Futures Support
+description: Support for using `Future[T]` and `Promise[T]` in REST Endpoints.
```

Comment:
I'm not sure we are using Markdown in the tooling everywhere, so I would remove the backticks.

Reply:
done